packages/core/helper-plugin/src/components/tests/RelativeTime.test.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import { lightTheme, ThemeProvider } from '@strapi/design-system';
+import { render as renderRTL, screen } from '@testing-library/react';
```

Comment:
if you import `render` from `@tests/utils` you can avoid a lot of the boilerplate you had to write :)